src/containers/SignIn/index.tsx

```diff
@@ -0,0 +1,97 @@
+import { t, Trans } from '@lingui/macro';
+import { Button, Segmented, Tooltip } from 'antd';
+import React from 'react';
+
+import logo from 'src/images/logo.svg';
+import { authClientConfigMap, ClientID, type SharedCredentials } from 'src/services/auth';
+
+import { SignInProps, useSignIn } from './hooks';
+import s from './SignIn.module.scss';
+import { S } from './SignIn.styles';
+
+// import {AppFooter} from '@beda.software/emr/dist/components/BaseLayout/Footer';
+
+interface SharedCredentialsProps {
+    sharedCredentials: SharedCredentials;
+}
+
+function SharedCredentials(props: SharedCredentialsProps) {
+    const { sharedCredentials } = props;
+
+    return (
+        <S.CredentialsWrapper>
```

Comment:
Replace two tabs with a list of options,
user can search for option,
Once option is selected login button is active.

<img width="1512" alt="Screenshot 2025-03-14 at 3 51 51 pm" src="https://github.com/user-attachments/assets/f678fc3b-e89c-4ea8-b8eb-923f04dab199" />